server/api/lib/trees/tree_crawler.js

```diff
@@ -86,14 +87,16 @@ const chooseCrawl = (crawlType) => {
 	const kkl = { 'crawler': crawlTreeExcelByFile, 'permitType': KKLTreePermit };
 	const jer = { 'crawler': crawlTreesHTML , 'permitType': JERTreePermit };
 	const rg  = { 'crawler': crawlRGTreesHTML , 'permitType': RGTreePermit};
+	const hh  = { 'crawler': crawlHHTreesHTML , 'permitType': HHTreePermit};
```

Comment:
The name `hh` is not obvious. I don't know why we are stinchy when it comes to names. It's hard to read `rg` and say - ah! it is Ramat-Gan. I think we can spare some character for a full name, e.g. `ramat_gan`, `hod_hasharon`.

Reply:
ok